docs/administering/config-file.md

```diff
@@ -51,10 +51,10 @@ typically a static publishing website, then serve a normal response.
 
 ### HTTPS_PORT
 
-A port number for Sandstorm to bind on and listen for HTTPS. Note that Sandstorm's built-in HTTPS
-assumes you are using the sandcats service, so that it can automatically renew a wildcard
-certificate for you. On a default install, if port 443 was available and the user chose to use
-sandcats, this is 443. If this config option is missing, Sandstorm's built-in HTTPS server is
+A port number for Sandstorm to bind on and listen for HTTPS. On a default install, if port 443
+was available and the user chose to use Sandcats, this is 443. However, this may be set for any
+Sandstorm-managed TLS configuration, including automated renewals of certificates with Let's Encrypt
+with a supported DNS provider. If this config option is missing, Sandstorm's built-in HTTPS server is
```

Comment:
I'm waffling on whether we should add something like "or a manually uploaded cert." This looks fine to me though; I'll leave it to you.

Reply:
I am kinda waffling on including automatic renewal here at all. It's probably not super relevant to the sandstorm.conf docs. The important part is "any Sandstorm-managed TLS configuration", but I guess I wanted to highlight this support since it's new?

I am counting on the ssl.md page to clarify all of the options and awesome... probably should stop procrastinating on tackling the big one. But I noticed this definitely inaccurate statement, and wanted to PR it before I forgot it was there again.

Reply:
SGTM; let's keep it as-is.

Reply:
Oh, uh... oops. b9ca01c is a thing now.

Reply:
That's cool too I guess.